src/GoogleAdWordsEventForwarder.js

```diff
@@ -124,23 +126,87 @@
+            }
+
+            if(customFlags[ENHANCED_CONVERSION_DATA + '.phone_number']) {
+                sanitizedConversionData.phone_number = customFlags[ENHANCED_CONVERSION_DATA + '.phone_number'];
+            }
+
+            if(customFlags[ENHANCED_CONVERSION_DATA + '.first_name']) {
+                sanitizedConversionData.first_name = customFlags[ENHANCED_CONVERSION_DATA + '.first_name'];
             }
-            if (enhancedConversionData.phone_number) {
-                window.enhanced_conversion_data.phone_number = enhancedConversionData.phone_number;
+
+            if(customFlags[ENHANCED_CONVERSION_DATA + '.last_name']) {
+                sanitizedConversionData.last_name = customFlags[ENHANCED_CONVERSION_DATA + '.last_name'];
+            }
+
+            if(customFlags[ENHANCED_CONVERSION_DATA + '.home_address.street']) {
+                home_address.street = customFlags[ENHANCED_CONVERSION_DATA + '.home_address.street'];
+            }
+
+            if(customFlags[ENHANCED_CONVERSION_DATA + '.home_address.city']) {
+                home_address.city = customFlags[ENHANCED_CONVERSION_DATA + '.home_address.city'];
+            }
+
+            if(customFlags[ENHANCED_CONVERSION_DATA + '.home_address.region']) {
+                home_address.region = customFlags[ENHANCED_CONVERSION_DATA + '.home_address.region'];
+            }
+
+            if(customFlags[ENHANCED_CONVERSION_DATA + '.home_address.postal_code']) {
+                home_address.postal_code = customFlags[ENHANCED_CONVERSION_DATA + '.home_address.postal_code'];
             }
-            if (enhancedConversionData.first_name) {
-                window.enhanced_conversion_data.first_name = enhancedConversionData.first_name;
+
+            if(customFlags[ENHANCED_CONVERSION_DATA + '.home_address.country']) {
+                home_address.country = customFlags[ENHANCED_CONVERSION_DATA + '.home_address.country'];
             }
-            if (enhancedConversionData.last_name) {
-                window.enhanced_conversion_data.last_name = enhancedConversionData.last_name;
+
+            if (Object.keys(home_address).length) {
+                sanitizedConversionData.home_address = home_address;
             }
-            if (enhancedConversionData.home_address) {
-                window.enhanced_conversion_data.home_address =
-                    enhancedConversionData.home_address;
+
+            return sanitizedConversionData;
+        }
+
+        function hasEnhancedConversionData(customFlags) {
+            return customFlags && Object.keys(customFlags).length && customFlags[ENHANCED_CONVERSION_DATA];
+        }
+
+        function hasStringEnhancedConversionData(customFlags) {
+            if (customFlags && Object.keys(customFlags).length) {
+                return Object.keys(customFlags)
+                    .toString()
+                    .includes(ENHANCED_CONVERSION_DATA);
```

Comment:
Even though it's more verbose, I would go with `indexOf(ENHANCED_CONVERSION_DATA) > -1` instead of `includes` for even more compatibility.  Until we get the proper transpiling in place, there's always the chance of a commonly used TV using an older chrome version which has bitten us in the past.